views/movies.hbs

```diff
@@ -0,0 +1,17 @@
+<h1>Ironhack Cinema</h1>
+<h2>Click on the movie to check the showtimes!</h2>
+<div id="movies" class="container">
+  {{#each movies}}
+    <div id="movieCard" class="card">
+      <div class="card__header">
```

Comment:
another nitpick -> try to keep your names consistent -> i.e. when you have movieCard make it cardHeader or movie_card...